README.md

```diff
@@ -10,6 +10,23 @@ This is a grants application for Allo. It is built using
 
 - [Node.js](https://nodejs.org/en/) (v18.x.x)
 - [Bun](https://bun.sh/) (v1.x)
+- Add env vars:
+
+Required
+```
+NEXT_PUBLIC_IPFS_READ_GATEWAY=
+NEXT_PUBLIC_PINATA_JWT=
+NEXT_PUBLIC_IPFS_READ_GATEWAY=
+NEXT_PUBLIC_IPFS_WRITE_GATEWAY=
+```
+
+Optional
+```
+NEXT_PUBLIC_ENVIRONMENT=
+ALCHEMY_ID=
+INFURA_ID=
+NEXT_PUBLIC_GRAPHQL_URL=
```

Comment:
^ this would be needed as well but we can add it more instructions on how to set that up in the following week